reactrestaurant/src/App.js

```diff
@@ -4,26 +4,34 @@ import './App.css';
 import Navbar from './components/navbar';
 import Footer from './components/footer';
 import Page from './components/page';
-
+import Accordian from './components/accordian';
+import './themestr.css';
+import NavTabs from './components/navtabs';
 /*
 navbar top
 carousel
   restuarant name
 */
-function App() {
-  return (
-    <>
-      <Navbar /> 
-      <Page />
-      <Footer />
-    </>
-    /*
-    state {0:home, 1:menu}
-    state == 0 ? jumbotron+carousel : menu, title, col, accordians, subtitle, dishes
-    */
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.props = {navItems:["Home","Menu","Directions"]};
```

Comment:
This appears to be your parent component, and you've created a this.props object, but props are objects that get passed to child components (i.e. Navbar, Page, etc. You will want to use a state object here.

https://reactjs.org/docs/state-and-lifecycle.html